src/html.sortable.js

```diff
@@ -610,7 +618,7 @@ var sortable = function (sortableElements, options) {
       startParent = this.parentElement
       startList = _serialize(startParent)
       // dispatch sortstart event on each element in group
-      _dispatchEventOnConnected(sortableElement, _makeEvent('sortstart', {
+      _dispatchEvent(sortableElement, _makeEvent('sortstart', {
```

Comment:
Sorry, but looking at it now, couldn't we just do `sortableElement.dispatchEvent( _makeEvent('sortstart', {…` instead if using the `_dispatchEvent` function? Sorry, I did not catch that before you wrote it.